samples/XAMLCatalog/XAMLCatalog/ProgrammaticViewController.m

```diff
@@ -41,8 +42,11 @@ - (void)viewDidLoad {
     // UIButton
     [self addMenuItemViewController:[[UIButtonViewController alloc] init] andTitle:@"UIButton"];
 
+    // UIButton
+    [self addMenuItemViewController:[[UIButtonWithControlsViewController alloc] init] andTitle:@"UIButton - controls"];
```

Comment:
Sure - ideally we'll revert update all existing VCs and keep the old naming convention (UISlider, UIButton, etc) but for now I can be more descriptive.

---
In reply to: [97406300](https://github.com/Microsoft/WinObjC/pull/1771#discussion_r97406300) [](ancestors = 97406300)